Documentation/getting-started.md

```diff
@@ -15,8 +15,9 @@ Configure the Edge, EdgeIdentity extensions in the mobile property using the Dat
 
 ----
 
-### Configure AEPEdgeMedia extension
-Currently AEPEdgeMedia doesn't have a Data Collection extension and needs to be configured programmatically.
+### Configure Media for Edge Network extension
```

Comment:
same question here, should we defer the configuration to the developer docs - e.g. https://raw.githubusercontent.com/adobe/aepsdk-edgeidentity-android/main/Documentation/getting-started.md

Reply:
Since the doc doesn't exist, how about we cleanup later?